Source/Fuse.Scripting.JavaScript/ModuleInstance.uno

```diff
@@ -67,16 +83,6 @@ namespace Fuse.Scripting.JavaScript
 			}
 		}
 
-		object EvaluateExports(Scripting.Context context)
-		{
-			EvaluateModule(context);
-
-			if (_moduleResult != null)
-				return _moduleResult.GetExports(context);
-
-			return null;
-		}
-
```

Comment:
It's not clear to me what all the refactoring in here is good for. I can't see any functional change, just moved code. Care to explain?

Reply:
The changes account for a simplification/split in the JavaScript tag into the implied one for model and the one the user explicitly does. In particular, the implied one can be persisted (moduleInstance) in Preview. These are all minor changes to accomodate that change.

Reply:
Ah, so `ReflectExports()` gets called by `ModelJavaScript`? That makes a lot of sense, thanks for the clarification :)